src/scene-entry-manager.js

```diff
@@ -167,6 +168,12 @@ export default class SceneEntryManager {
     this.scene.emit("username-changed", { username: displayName });
   };
 
+  _setupKicking = () => {
+    document.body.addEventListener("kicked", ev => {
+      NAF.connection.entities.removeEntitiesOfClient(ev.detail.clientId);
```

Comment:
I'm guessing this is only going to remove their owned objects, which seems OK for now -- but I think we probably want to do a pass where we start capturing the client id of the user who created the object, and remove the objects the kicked user created. (Obvious, but if you touch a kicked user's object, it still probably makes sense to remove it if they are kicked, and alternatively if they touch your objects before they are kicked probably doesn't make sense to remove them.)